### PR TITLE
Benchmark: React testing and charting

### PR DESCRIPTION
**Pull request by chrisbolin**
- add dummy react element test
- remove slower packages from benchmark. no need to test against so many, just the closest competitors
- benchmark generic and react functionality and output data to console and JSON (for chart)

### CODE REVIEW

**Comment by chrisbolin** [comment]
CI failed because Node 4 doesn't support the `...spread` syntax. I'll fix that

**Comment by chrisbolin** [comment]
the benchmarking improvements let us make a chart for the README

image

File: spec/tests.js [hunk 1]
```diff
@@ -318,3 +318,53 @@ module.exports = [
...
+        equal: false
+      }
+    ]
+  }
+];
+
+module.exports = {
+  generic,
+  react,
+  all: generic.concat(react)
```
Comment (ryan-roemer) [inline spec/tests.js]: NIT: This is a personal preference, but like:

```js
[].concat(generic, react)
```

a little better for readability...

Comment (chrisbolin) [reply spec/tests.js]: yeah i like that better, too. More inline with the ES6 `[...generic, ...react]` I had before

**Pull request merged**